templates/project/.github/workflows/publish.yml

```diff
@@ -0,0 +1,60 @@
+name: publish
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build-project:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/viamrobotics/micro-rdk-dev-env:amd64
+    steps:
+    - name : Checkout main branch code
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 2
+    - name: Build esp32 OTA binary
+      run: |
+        bash -c '. /home/testbot/.bash_profile ; git config --global --add safe.directory "$ESP_ROOT"/esp-idf && make build-esp32-ota'
+        cp target/xtensa-esp32-espidf/project-ota.bin project-ota.bin
+    - name: Upload release Lib
+      uses: actions/upload-artifact@v4
+      with:
+        name: ota-binary
+        path: |
+          project-ota.bin
+
+  publish-release:
+    needs: [build-project]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check release type
+      id: check-tag
+      run: |
+        if echo ${{ github.event.ref }} | grep -Eq '^refs/tags/v.*rc[0-9]{1}$'; then
```

Comment:
should we add a project README where we can add a line about this GH actions workflow?

Reply:
I think not, we don't want that README to end up in the generated template because owners of the projects will probably want their own READMEs. If anything, details about how to use the project template should be in DEVELOPMENT.md. But I don't want to include it until the github links are usable in OTA and we can successfully document a complete release process.